src/glfw3.jl

```diff
@@ -509,9 +516,12 @@ GLFW.SetWindowIcon(win, buff)
 GLFW.PollEvents() # seems to need a poll events to become active
 ```
 """
-SetWindowIcon(window::Window, image::Matrix{NTuple{4, UInt8}}) = ccall((:glfwSetWindowIcon, lib),
+SetWindowIcon(window::Window, image::AbstractMatrix{NTuple{4,UInt8}}) = ccall((:glfwSetWindowIcon, lib),
```

Comment:
I suggest getting rid of the `ccall` and just calling the new plural variant:
```
SetWindowIcon(window, image) = SetWindowIcon(window, [image])
```

Reply:
`ERROR: LoadError: GLFWError (PLATFORM_ERROR): Win32: Failed to create RGBA bitmap`

when trying your suggestions and passing only when icon

I will drop the last commit

Reply:
resolved